PiggyBank.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" type="text/css" href="PiggyBank.css" media="screen" />
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="PiggyBank.js"></script>
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js" integrity="sha256-R4pqcOYV8lt7snxMQO/HSbVCFRPMdrhAFMH+vr9giYI=" crossorigin="anonymous"></script> -->
+    <meta charset="utf-8">
+    <title>My Piggy Bank</title>
+  </head>
+  <body>
```

Comment:
Everything aside from buttons seem to be using only #id's for CSS.  IMO this is backwards from standard convention and doesn't make for great options when declaring CSS.  You lose your chance to focus on specificity when styling.